pkg/ssh/connection_native.go

```diff
@@ -180,3 +181,56 @@ func nativeConnectionScp(options ConnectionScpOptions) (*ConnectionScpReport, er
 
 	return &ConnectionScpReport{Response: remotePath}, nil
 }
+
+func nativeConnectionDial(options ConnectionDialOptions) (*ConnectionDialReport, error) {
+	dst, uri, err := Validate(options.User, options.Host, options.Port, options.Identity)
+	if err != nil {
+		return nil, err
+	}
+	_, err = ValidateAndConfigure(uri, options.Identity, options.InsecureIsMachineConnection)
+	if err != nil {
+		return nil, err
+	}
+
+	ssh, err := exec.LookPath("ssh")
+	if err != nil {
+		return nil, fmt.Errorf("no ssh binary found")
+	}
+
+	output := &bytes.Buffer{}
+	errors := &bytes.Buffer{}
+	if strings.Contains(uri.Host, "/run") {
+		uri.Host = strings.Split(uri.Host, "/run")[0]
+	}
+
+	sshArgs := []string{uri.User.String() + "@" + uri.Hostname(), "podman", "info"}
```

Comment:
Why would common output podman commands?  If this data is needed, it should be a param of the function,
Or this function should be in podman